autopts/pybtp/btp/bap.py

```diff
@@ -130,12 +130,13 @@ def bap_send(ase_id, data_ba, bd_addr_type=None, bd_addr=None):
 def bap_broadcast_source_setup(
         streams_per_subgroup, subgroups, coding_format, vid, cid,
         codec_ltvs, sdu_interval, framing, max_sdu, retransmission_number,
-        max_transport_latency, presentation_delay):
+        max_transport_latency, presentation_delay, broadcast_id=0):
```

Comment:
Did PTS have an issue with broadcast_id == 0? If so, then I don't think we should default the braodcast_id to 0, and expect always expect the caller to provide a value here

Reply:
Ok, changed to '0x123456'